src/cloudant/database.py

```diff
@@ -630,18 +630,16 @@ def __iter__(self, remote=True):
         if not remote:
             super(CouchDatabase, self).__iter__()
         else:
-            next_startkey = '0'
+            next_startkey = u'\u0000'
```

Comment:
I think this is worthy of a comment, it is not obvious what the purpose is to someone not intimately familiar with doc sorting.